README.md

```diff
@@ -34,9 +34,9 @@ Note that the grammar is written completely in JavaScript (`grammar.js`), the ot
 
 ### Testing
 
-To run the unit tests, run:
+To run the parser tests, run:
 
->`npm run test`
+>`npm x -- tree-sitter test`
```

Comment:
If we keep the scripts we can even shorten this:

```suggestion
>`npm test`
```

Reply:
But then we will lose the node package tests.

Reply:
Then we should adapt the `test` script to run `tree-sitter test` (too?)

Reply:
I'd rather add a different script for it.